bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -566,6 +566,10 @@ impl RoomListItem {
         self.inner.init_timeline_with_builder(timeline_builder).map_err(RoomListError::from).await
     }
 
+    async fn is_encrypted(&self) -> bool {
```

Comment:
Please add a doc comment, documentation is nowadays visible over the FFI, also please document that the correctness of this might depend on specifying `m.room.encryption` as required state.

Reply:
Let me know if https://github.com/matrix-org/matrix-rust-sdk/pull/3602/commits/0162f6ba49e92264416d45dc6a99f77d7d62f0d3 works for you.